### PR TITLE
Created buttons in navbar and "saved" message 

### PR DESCRIPTION
**Pull request by petamar**
Created buttons in navbar and "saved" message

### CODE REVIEW

**Comment by max-romagnoli** [comment]
This PR closes #45 #19

**Comment by max-romagnoli** [comment]
Once PR #49 is merged in main we can work on integrating this PR with main. There will be probably be some conflicts with the new changes to `Button.js` I made in PR #49.
A further development of Issue #19 would be to actually create the functioning Links to other pages. That though will be for another branch, in my opinion (perhaps when developing Issue #53).

**The aim is to close this PR before Tuesday**, with all the infrastructure in place to start working on multi-page functionalities.
@ciannamacmahon @MaxCunningham19 we would like your input on this.

File: client/src/components/Navbar.js [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
+import { Button } from './Button'
 import styles from './Navbar.module.css'
 
 function Navbar() {
   return (
     <>   
         <nav className={styles['Navbar']}>
             <div className={styles['navbar-container']}>
+            <div className={styles['empty-container']}>
+              </div>
```
Comment (MaxCunningham19) [inline client/src/components/Navbar.js:11]: what's the point of this empty container?

File: client/src/components/Navbar.js [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
+              <div className={styles['buttons-container']}>
+                <Button buttonStyle="navbar-plain" text=" 🗎 My Documents"></Button>
 
+              </div>
+              <div className={styles['buttons2-container']}>
+                <Button buttonStyle="navbar-plain" text=" 🗀 Browse"></Button>
+
+              </div>
+              <div className={styles['empty-container']}>
+              </div>
```
Comment (MaxCunningham19) [inline client/src/components/Navbar.js:21]: same here

**Review by MaxCunningham19 — APPROVED** [review]
LGTM

File: client/src/components/Navbar.js (hunk 1 above)
Comment (max-romagnoli) [reply client/src/components/Navbar.js:11]: Yeah all these divs are unnecessary. We only need one `"navbar-container"` and the two `<Button />` inside it. The flex container then should handle the alignment on its own.

**Review by ciannamacmahon — APPROVED** [review]
yep looks all good

**Pull request merged**